ptbcontrib/postgres_persistence/README.md

```diff
@@ -0,0 +1,35 @@
+# Persistence class with Postgresql database as backend.
+
+* How to use:
```

Comment:
And here that you do a list item without indenting it. Id suggest making it a level 2 header.
```suggestion
## How to use:
```